release/release_tests.yaml

```diff
@@ -1936,6 +1936,34 @@
 
   alert: default
 
+########################
+# Train tests
```

Comment:
Going forward, what tests should belong in `train_tests` and which tests should belong in `ml_user_tests/train`?

Reply:
ML user tests have a specific set of properties that we want to test https://github.com/ray-project/ray/blob/ray-1.11.0/release/.buildkite/build_pipeline.py#L268.

Unfortunately this comment got removed during the refactor.